yarn.lock

```diff
@@ -4,19 +4,19 @@
 
 "@babel/code-frame@^7.0.0":
   version "7.12.13"
-  resolved "https://registry.yarnpkg.com/@babel/code-frame/-/code-frame-7.12.13.tgz#dcfc826beef65e75c50e21d3837d7d95798dd658"
+  resolved "https://registry.npmjs.org/@babel/code-frame/-/code-frame-7.12.13.tgz"
```

Comment:
Is this what happens when you run `npm install` instead of `yarn install`? I would guess this change should be excluded from the PR.

Reply:
🤷🏻 I would assume that npm install wouldn't touch a yarn.lock at all